packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -481,6 +481,68 @@ void main() {
     final BoxDecoration decoration = decoratedBox.decoration;
     expect(decoration.border, isNull);
   });
+
+  testWidgets('Standard title golden', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new WidgetsApp(
+        color: const Color(0xFFFFFFFF),
+        onGenerateRoute: (RouteSettings settings) {
+          return new CupertinoPageRoute<void>(
+            settings: settings,
+            builder: (BuildContext context) {
+              return const RepaintBoundary(
+                child: const CupertinoPageScaffold(
+                  navigationBar: const CupertinoNavigationBar(
+                    middle: const Text('Bling bling'),
+                  ),
+                  child: const Center(),
+                ),
+              );
+            },
+          );
+        },
+      ),
+    );
+
+    await expectLater(
```

Comment:
Was discussing this with @Hixie.  For now at least, we'll skip the golden file tests on non-Linux platforms (and require that developers update their goldens while running on Linux).

So tldr: add a `skip: !Platform.isLinux` to these new `testWidgets()` methods

Reply:
cool, thanks for following up. I'll update the goldens